src/pages/instances/InstanceList.tsx

```diff
@@ -94,6 +99,7 @@ const InstanceList: FC = () => {
   const [sizeHidden, setSizeHidden] = useState<string[]>([]);
   const [selectedNames, setSelectedNames] = useState<string[]>([]);
   const [processingNames, setProcessingNames] = useState<string[]>([]);
+  const [migratingInstances, setMigratingInstances] = useState<string[]>([]);
```

Comment:
I think we should not introduce additional local state for this. Can we rely on the global `instanceLoading` state instead, to check if an instance is currently migrating?

Reply:
to use `instanceLoading`, we would need to handle setting individual instance states at the API level i.e. in `migrateInstanceBulk`, and we would also have to update `instanceLoading.getType` to take in a string and not `LxdInstance`. 

I think better if we just use `processingNames` instead of creating a new state `migratingInstances`? I'm already setting `processingNames` on starting bulk migration. Note `processingNames` are cleared after migration is complete, regardless of successful completion of individual operations.

Reply:
Okay, sounds good to me.